content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website looks like, but everyone can interact with it. This principle is the idea behind XMPP for instant messaging.
+
+We at the XMPP Standards Foundation, but also many others involved in this open technology, think that XMPP is a good choice as a communication technology when it comes to this question. XMPP as a protocol has been around for about two decades since its creation in 1999 and a lot of experience has been gathered during that period. Mainly, this includes several important key indicators for the choice of technology and enabling of sovereignty as 
+there are:
+
+- Decentralization (federation) of communication services
+- Standardization and extensibility of technology
+- Innovation and the use of open development
+- Privacy and control, also by trustful encryption
+- Enabling of free speech and freedom of software
+
+> XMPP provides defined information how to handle the communication in a network. The user decides which messaging software suits or where the data should be stored.
+
+In contrast to the current situation of many people suggesting one specific messenger, in the XMPP universe you have the choice between many messengers – pick the one you like and not what others like (and maybe think that you might like as well) until the next hype starts. The difference is that you still have the comfort to be in the same network, independent of your choice today. I think this is a real solution.
+
+### It is not a choice of the messenger, but of the technology
+
+Consider a decision on the technology you intend to use, then decide which messenger suits you or your environment. XMPP and its community have a long-lasting experience, which has been successfully utilized in a lot of software. We think this was and still is a good solution for the entire internet and most people and organizations out there – and if not yet, it is open for you to make modifications in the future to make it do what you need.
+
+> XMPP has not just a wide range of messaging software; there is also various server software and the tools (libraries) to build infrastructure yourself.
+
+There is not one single solution how communication is designed. Make a choice of the technology which is a persistent and sustainable decision. For that purpose and many more we recommend the use of XMPP – the Extensible Messaging and Presence Protocol.
```

Comment:
"There is not a single way in which communication apps are designed" maybe.

Reply:
Yes